src/rgb/validation.rs

```diff
@@ -205,7 +207,7 @@ pub enum Info {
     UncheckableConfidentialStateData(NodeId, usize),
 }
 
-pub struct Validator<'validator> {
+pub struct Validator<'validator, R: TxResolver> {
```

Comment:
This will probably break RGB-node code

Reply:
Yes, but I've already opened lnp-bp/rgb-node#13 that applies the required changes and also adds a proper resolver based on rust-electrum-client

Reply:
sorry, haven't seen it